Add bigquery hooks (#779)

beckjake opened the pull request:
Add pre/post hook support for bigquery (fixes #779).

drewbanin commented on test/integration/014_hook_tests/test_model_hooks_bq.py:
```diff
@@ -0,0 +1,146 @@
+            'models': {},
+            'seeds': {
+                'post-hook': [
+                    'alter table {{ this }} add column new_col int',
```
does this work? `alter table ... add column` is not supported on BigQuery! I think a better test might just update a column or insert a record

beckjake replied:
It turns out that we weren't loading the seed file on Postgres or BigQuery because the extension is `.sql` instead of `.csv`, so these hooks never run and the tests always pass.

Good stuff.

beckjake commented:
@drewbanin Per our in-person/slack discussion, fixed the hook + seed issues you found and added tests. Let me know when you've tested this out and if it's good to go I'll merge it

drewbanin commented:
@beckjake just gave this a spin locally and it works great! Feel free to merge when the tests pass
